packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -0,0 +1,413 @@
+import { useState, useCallback, useMemo, forwardRef } from 'react';
```

Comment:
The underline below Upload should change based on the white-labelled brand color, right?
<img width="583" alt="Screenshot 2024-02-29 at 3 41 57 PM" src="https://github.com/razorpay/blade/assets/24487274/aae92585-b555-4542-aec0-2c4511d16f25">

Reply:
We need to fix it for our border tokens, card has the same issue.
<img width="1234" alt="Screenshot 2024-02-29 at 5 42 21 PM" src="https://github.com/razorpay/blade/assets/46647141/127a34d9-e7dd-4e84-86f3-822cb9f670d5">